packages/zowe-explorer/scripts/downloadLocalizations.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+/**
+ * Downloads the latest translations submitted to Artifactory to
+ * https://zowe.jfrog.io/ui/repos/tree/General/libs-snapshot-local/org/zowe/vscode/zowe-explorer-g11n
+ * and copies them into the provides them into the top-level and l10n for building a localized extension.
+ */
+
+const path = require("path");
+const fs = require("fs");
+const process = require("process");
+const tmp = require("tmp");
+const admZip = require("adm-zip");
+
+const artifactPath = "libs-snapshot-local/org/zowe/vscode/zowe-explorer-g11n";
+const baseUrl = new URL("https://zowe.jfrog.io");
+const getLatestInfoApiUrl = new URL(`artifactory/api/storage/${artifactPath}?lastModified`, baseUrl);
+
+(async () => {
+    const listResponse = await fetch(getLatestInfoApiUrl)
+        .then((r) => r.json())
+        .catch((error) => {
```

Comment:
if possible, please keep a consisten style for handling promises
- `async/await` with proper `try/catch`
- `.then/.catch`

Reply:
I literally copied that from the other scripts you had :-)

Reply:
> I literally copied that from the other scripts you had :-)

Thanks for sharing!  🙏 

I just hoped to have some consistency going forward, but we can revisit these scripts at a later time.

Reply:
I refactored the code getting rid of the thens.